tools/testing-server/plugins/agent-injector/script-transform.js

```diff
@@ -50,9 +50,21 @@ module.exports = function (request, reply, testServer) {
 
       // Replace {script-injection}
       if (chunkString.indexOf('{script-injection}') > -1) {
+        let scriptContent = ''
+        if (testServer.config.polyfills && request.query.scriptString) { // if the test is running in IE11, the test scriptString may need Babel transpilation
```

Comment:
Shouldn't need to check for scriptString in the query object. The logic check can be a little confusing to look at for those not familiar with the code.

Reply:
```
const scriptStream = Readable.from(Buffer.from(request.query.scriptString, 'base64'))
```
Have trouble handling empty strings